Introducting LLM debug mode (#12)
speechmatics/speechmatics-flow: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

> ## [0.0.10] - 2025-02-13

Reviewer: I know we're not doing a great job at following semantic versioning rules, but this time I would bump to 0.1.0 as it's introducing support for a new feature, not just bugfixing existing code.

Author: done in https://github.com/speechmatics/speechmatics-flow/pull/12/commits/d22553cbbc50f64b10bf2780a28e6620e706c974

### Added

- Added Debug mode
  - Introduced a new debug field in `StartConversation`.

>   - When debug mode is enabled, the client can receive LLM interaction messages exchanged between the flow service and LLM backends.

Reviewer: ```suggestion
  - When LLM debug mode is enabled, the client will receive debug messages with LLM request and response content
```
I would rephrase with something more like that
